content/howto/geodesic/build-your-own-toolbox.md

```diff
@@ -27,7 +27,7 @@ RUN apt-get install -y your-needed-package
 docker build . -t acme:latest
 
 # Install on your machine as your own executable toolbox
-docker run --rm acme:latest | APP_NAME=acme bash -s latest-debian
+docker run --rm acme:latest init | APP_NAME=acme bash -s latest-debian
```

Comment:
This was wrong before, has been wrong for a long time, let's not change it like was right and we are fixing it, because even with this change it is wrong. I think the following MIGHT work, but it might not and I don't have time to check it now.
```suggestion
docker run --rm acme:latest init | APP_NAME=acme bash -s latest
```

If you want to build your own customized Geodesic, the recommended way is to make a Dockerfile as this note sketches out (although without any of the AWS_VAULT stuff) and then copy and customize the Geodesic `Makefile` (setting variables at the top) to build and install it with `make build`, `make install` and/or `make all`.

@osterman Really, we probably should publish a Makefile template because the Geodesic Makefile is not quite right for derived versions.

Reply:
I am including Makefile and Dockerfile templates in latest [Geodesic PR](https://github.com/cloudposse/geodesic/pull/727)